contracts/src/v0.8/dev/interfaces/OCR2DRClientInterface.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
+
+/**
+ * @title OCR2DR client interface.
+ */
+interface OCR2DRClientInterface {
+  /**
+   * @notice OCR2DR response handler called by the designated oracle.
+   * @param requestId The requestId returned by OCR2DRClient.sendRequest().
+   * @param response Aggregated response from the user code.
+   * @param err Aggregated error from the user code, or an error string from the execution pipeline.
```

Comment:
Re-word to be clear that the error is always OCR-aggregated, even if it's coming from the execution pipeline.

Reply:
Agreed, fixed.